.docker/Dockerfile

```diff
@@ -1,10 +1,21 @@
-FROM node:7.4.0
```

Comment:
I've tried to use one with Docker, but it is based on Alpine and Docker Compose did not work there. So I took base Ubuntu image instead. I guess this one could be use as well, but I don't see any pros or cons.